Harvest.Web/Services/EmailService.cs

```diff
@@ -25,13 +27,18 @@ public interface IEmailService
         Task<bool> ApproveAccounts(Project project, string[] emails);
 
         Task<bool> InvoiceExceedsQuote(Project project, decimal invoiceAmount, decimal quoteRemaining);
+        Task<bool> NewTicketCreated(Project project, Ticket ticket);
+        Task<bool> TicketReplyAdded(Project project, Ticket ticket, TicketMessage ticketMessage);
+        Task<bool> TicketAttachmentAdded(Project project, Ticket ticket, TicketAttachment[] ticketAttachments);
+        Task<bool> TicketClosed(Project project, Ticket ticket);
     }
 
     public class EmailService : IEmailService
     {
         private readonly AppDbContext _dbContext;
         private readonly IEmailBodyService _emailBodyService;
         private readonly INotificationService _notificationService;
+        public const string BaseUrl = "https://harvest.caes.ucdavis.edu";

```

Comment:
Maybe have this as an option, so that we can configure it in secrets to point to localhost?

Reply:
Yeah. Just being lazy for now.